app_dart/lib/src/model/appengine/task.dart

```diff
@@ -353,7 +353,7 @@ class Task extends Model<int> {
       case Result.success:
         return status = statusSucceeded;
       case Result.canceled:
-        return status = statusCancelled;
+        return status = statusInfraFailure;
```

Comment:
Please add the Task.Cancelled state here https://cs.opensource.google/flutter/cocoon/+/main:app_dart/lib/src/service/build_status_provider.dart;l=140. That way we can keep the cancelled state in the task model.

Reply:
Good catch. Updated.